truewiki/storage/local.py

```diff
@@ -37,7 +37,7 @@ def get_repository_url(self):
         return ""
 
     def get_file_nonce(self, filename: str) -> str:
-        return str(os.path.getmtime(f"{self._folder}/{filename}"))
+        return str(os.path.getmtime(f"{self._folder}/{filename}")) if self.file_exists(filename) else "0"
```

Comment:
You sure `filename` is correct? Shouldn't that also be `f"{self._folder}/{filename}"`? I think it now always returns "0" doesn't it?

Reply:
Ah, good catch. I blame github copilot (just turned it on today) XD

Reply:
Well I _did_ test the changes, but on an old fix. I first fixed this by wrapping the calls in `source` and `web_routes` but realized it would be better to do it inside the singleton, because other backends (git) use a commit hash as the nonce which doesn't need this check.

Fixed now!